src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/store/rca/HighHeapUsageClusterRca.java

```diff
@@ -170,15 +175,12 @@ public ResourceFlowUnit operate() {
     }
   }
 
+  /**
+   * This is a cluster level RCA vertex which by definition can not be serialize/de-serialized
+   * over gRPC.
+   */
   @Override
   public void generateFlowUnitListFromWire(FlowUnitOperationArgWrapper args) {
-    final List<FlowUnitMessage> flowUnitMessages =
-        args.getWireHopper().readFromWire(args.getNode());
-    List<ResourceFlowUnit> flowUnitList = new ArrayList<>();
-    LOG.debug("rca: Executing fromWire: {}", this.getClass().getSimpleName());
-    for (FlowUnitMessage flowUnitMessage : flowUnitMessages) {
-      flowUnitList.add(ResourceFlowUnit.buildFlowUnitFromWrapper(flowUnitMessage));
-    }
-    setFlowUnits(flowUnitList);
+    LOG.error("RCA: {} should not be send over from network", this.getClass().getSimpleName());
```

Comment:
I am thinking we throw an exception and capture the stacktrace if this is unexpected ?

Reply:
done